### ISSUE TITLE
Dual licenses

### ISSUE THREAD

**Issue by Kigaard** [bio-tools/biotoolsSchema#209]
Some tools seems to have two (open-source) licenses.

It should be noted that this is only the case for a hand-full of tools.

**Comment by matuskalas** [comment]
Good point @Kigaard! We need the possibility to record multiple licenses.
